src/benchmarks/micro/corefx/System.Text.Json/Utf8JsonWriter/Perf.Base64.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Buffers;
+using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Extensions;
+using MicroBenchmarks;
+
+namespace System.Text.Json.Tests
+{
+    [BenchmarkCategory(Categories.CoreFX, Categories.JSON)]
+    public class Perf_Base64
+    {
+        private ArrayBufferWriter<byte> _arrayBufferWriter;
+        private byte[] _data;
+
+        [Params(10, 100, 1000)]
+        public int NumberOfBytes { get; set; }
+
+        [Params(true, false)]
+        public bool Formatted;
+
+        [Params(true, false)]
+        public bool SkipValidation;
+
+        [GlobalSetup]
+        public void Setup()
+        {
+            _arrayBufferWriter = new ArrayBufferWriter<byte>();
+            _data = ValuesGenerator.Array<byte>(NumberOfBytes);
+        }
+
+        [Benchmark]
+        public void WriteByteArrayAsBase64()
+        {
+            _arrayBufferWriter.Clear();
+            using (var json = new Utf8JsonWriter(_arrayBufferWriter, new JsonWriterOptions { Indented = Formatted, SkipValidation = SkipValidation }))
+            {
+                json.WriteBase64String("data", _data);
```

Comment:
`WriteBase64String` isn't available in .NET Core SDK `3.0.100-preview6-012162` (latest right now). Wasn't that API picked up by the SDK?

Reply:
I believe we should be picking up the latest available `3.0.100-preview7-012258`: https://dotnetcli.blob.core.windows.net/dotnet/Sdk/master/latest.version

The CI seems to be ok with it. I am not sure where this version is coming from since I am not totally familiar with how dotnet/performance updates the SDK it uses.

Are you hitting this issue locally?

**Can you check the System.Text.Json.dll in ildasm and see if the API is visible in your version of the SDK** (under `<SDK root>\shared\Microsoft.NETCore.App\3.0.0-preview7-27803-11\System.Text.Json.dll`)?

For the newer version I listed above, I do see it:
![image](https://user-images.githubusercontent.com/6527137/58926365-d338ac80-86ff-11e9-97d4-7ef9ed6ec22c.png)

You could also try running the script to fetch the latest SDK:
https://github.com/dotnet/performance/blob/81cbc1fe3c14ace0bfc3e220472f2fea4bcbca50/docs/benchmarking-workflow-corefx.md#running-against-the-latest-net-core-sdk
```text
py.exe .\scripts\benchmarks_ci.py -f netcoreapp3.0
```

cc @billwert, @jorive

Reply:
> I just wondered as the corresponding PR was merged 4 days back, so it should be in the latest SDK (by yesterday).

Sometimes it takes a bit longer for the bits to flow up to the SDK. That's probably why the API wasn't available. Thanks for providing the details.